termbox/compat.go

```diff
@@ -25,17 +25,32 @@ import (
 var screen tcell.Screen
 var outMode OutputMode
 
+func GetScreen() tcell.Screen {
+	return screen
+}
+func SetScreen(s tcell.Screen) {
+	screen = s
+}
+
 // Init initializes the screen for use.
 func Init() error {
+	if screen != nil {
+		// only init once
+		return nil
+	}
 	outMode = OutputNormal
-	if s, e := tcell.NewScreen(); e != nil {
```

Comment:
Why the restructuring in this function?

Reply:
slightly clearer. Not strictly necessary, but a little easier to follow.

Reply:
I actually prefer the old syntax.